app/facades/paginate_facade.rb

```diff
@@ -0,0 +1,33 @@
+class PaginateFacade
+
+  def get_gutenberg_book(guten_id)
+    JestamouseService.new.get_book_text(guten_id)
+  end
+
+  def remove_extra_returns(raw_text)
+    raw_text.gsub(/(\S)\r\n(\S){1}/, '\1 \2')
+  end
+
+  def paginate_text(clean_text)
+    paginated_book = []
+    until clean_text.empty?
+      text_set = clean_text.truncate(1000, separator: /\s/, omission:'')
+      paginated_book.push(text_set)
```

Comment:
Line 14 - change to: `text_set = clean_text.truncate(600, separator: /\s/, omission: ''), 1000 characters per page is too many. We were aiming between 500-700, so lets see how 600 looks to start
Line 15 - change to: `paginated_book.push(text_set.lstrip)` to remove the space at the beginning of each new page

Reply:
Update. I like the .lstrip added. Character count now set to 600, instead of 1000